packages/discovery-provider/src/challenges/challenges.json

```diff
@@ -137,5 +137,16 @@
     "starting_block": 220157041,
     "weekly_pool": 25000,
     "cooldown_days": 7
+  },
+  {
+    "id": "os",
+    "name": "ONE_SHOT",
+    "type": "aggregate",
+    "amount": 1,
+    "active": true,
```

Comment:
should we set these to false for now?

Reply:
i think it's safe to leave it but ill just set prod to false, just gotta remember to turn it back on